content/file-and-directory-naming/file-directory-naming-assignment/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+content_type: project
+flavours:
+  - java
+  - javascript
+  - markdown
+  - python
+prerequisites:
+  hard:
+    - file-and-directory-naming/general
+protect_main_branch: false
+ready: true
+submission_type: repo
+title: File and Directory Naming Assignment
+---
+
+Please answer the following questions in markdown files and submit following the instructions below.
+
+## Questions
+
+1. Here are some directory names that follow different naming conventions: Foo_Bar, foo_bar, FooBar, fooBar. Which of the directory names follows the convention used in your language?
+
+2. Here are some file names that follow different naming conventions: MyFile, my_file, My_File. Which of the file names follows the convention used in your language?
+
+3. Here are function names that follow different naming conventions: myFunction, my_function, MyFunction, My_function, My_Function. Which of the function names follows the convention used in your language?
+
+4. Why is it important to use the recommended naming convention?
+
+## How to submit your work
+
+Please follow the following instructions to submit your work:
+
+{{< contentlink path="project-submission-instructions/markdown-questions" >}}
```

Comment:
This link gets rendered like this:
![image](https://github.com/Umuzi-org/ACN-syllabus/assets/54069197/059e1995-a943-4ce8-b943-3ce01df64542)

I assume it's a problem. See if you can't find a way to remove that, otherwise creating a separate ticket for this sort of issue might be the way to go.

Reply:
@Ngoakor12 it will need a separate ticket, I have seen that [TODO] it many other places.